lib/fpm/command.rb

```diff
@@ -346,6 +351,22 @@ def execute
       end
     end
 
+    # If --exclude-file was specified, read it as a file and append to
+    # the exclude pattern list.
+    if !exclude-file.nil?
```

Comment:
You probably meant `exclude_file` not `exclude-file` (probably confusing if you're looking at the flag name, but `exclude-file` in ruby means 'subtract `file` from `exclude` variable')